monitoring/definitions/postgres.go

```diff
@@ -67,15 +67,23 @@ func Postgres() *monitoring.Dashboard {
 				Rows: []monitoring.Row{
 					{
 						monitoring.Observable{
-							Name:              "postgres_up",
-							Description:       "database availability",
-							Owner:             monitoring.ObservableOwnerDevOps,
-							DataMustExist:     false, // not deployed on docker-compose
-							Query:             "pg_up",
-							Panel:             monitoring.Panel().LegendFormat("{{app}}"),
-							Critical:          monitoring.Alert().LessOrEqual(0).For(5 * time.Minute),
-							PossibleSolutions: "none",
-							Interpretation:    "A non-zero value indicates the database is online.",
+							Name:          "postgres_up",
+							Description:   "database availability",
+							Owner:         monitoring.ObservableOwnerDevOps,
+							DataMustExist: false, // not deployed on docker-compose
+							Query:         "pg_up",
+							Panel:         monitoring.Panel().LegendFormat("{{app}}"),
+							Critical:      monitoring.Alert().LessOrEqual(0).For(5 * time.Minute),
+							// Similar to ContainerMissing solutions
+							PossibleSolutions: fmt.Sprintf(`
+								- **Kubernetes:**
+									- Determine if the pod was OOM killed using 'kubectl describe pod %[1]s' (look for 'OOMKilled: true') and, if so, consider increasing the memory limit in the relevant 'Deployment.yaml'.
+									- Check the logs before the container restarted to see if there are 'panic:' messages or similar using 'kubectl logs -p %[1]s'.
```

Comment:
usually, the oom reaper kills one of the postgres child processes instead of killing the main process when it's under pressure, hence the OOMKILLED status is rarely reflected on the container status.

```suggestion
									- Check the logs before the container restarted to see if there are 'panic:' messages or similar using 'kubectl logs -p %[1]s'.
									- Check if there is any OOMKILL event using the provisioning panels
									- Check kernel logs using `dmesg` for OOMKILL events on worker nodes
```